dask/dataframe/tests/test_groupby.py

```diff
@@ -433,8 +433,10 @@ def test_groupby_get_group():
 
 
 @pytest.mark.xfail(
-    sys.version_info[0] == 3 and sys.version_info[1] == 6,
-    reason="nlargest() behaves differently with an older version of pandas on Python 3.6 "
+    sys.version_info[0] == 3
+    and sys.version_info[1] == 6
+    and PANDAS_VERSION == "0.23.4",
```

Comment:
This is OK, since pandas won't release any more for py36 (right, @TomAugspurger ?). Might be cleaner with a relative or LooseVersion comparison.